src/blueapi/config.py

```diff
@@ -49,6 +49,14 @@ class WorkerEventConfig(BlueapiBaseModel):
     broadcast_status_events: bool = True
 
 
+class MetadataConfig(BlueapiBaseModel):
+    # TODO: Reconcile data_session (bluesky term) with instrument_session (DLS term)
+    data_session: str = "aa123456"
+
+    # TODO: Does/should this conflict with the ${BEAMLINE} environment variable
```

Comment:
I think this can be safely merged and turned into an issue

Reply:
https://github.com/DiamondLightSource/blueapi/issues/866